zds_schema/constants.py

```diff
@@ -67,3 +67,93 @@ class RolTypes(DjangoChoices):
 class ObjectTypes(DjangoChoices):
     besluit = ChoiceItem('besluit', _("Besluit"))
     zaak = ChoiceItem('zaak', _("Zaak"))
+
+
+
+class Archiefnominatie(DjangoChoices):
+    blijvend_bewaren = ChoiceItem('blijvend_bewaren',
+        _("Het zaakdossier moet bewaard blijven en op de Archiefactiedatum "
+          "overgedragen worden naar een archiefbewaarplaats.")
+    )
+    vernietigen = ChoiceItem('vernietigen',
+        _("Het zaakdossier moet op of na de Archiefactiedatum vernietigd "
+          "worden.")
+    )
+
+class Archiefstatus(DjangoChoices):
+    nog_te_archiveren = ChoiceItem('nog_te_archiveren',
+        _("De zaak cq. het zaakdossier is nog niet als geheel gearchiveerd.")
+    )
+    gearchiveerd = ChoiceItem('gearchiveerd',
+        _("De zaak cq. het zaakdossier is als geheel niet-wijzigbaar "
+          "bewaarbaar gemaakt.")
+    )
+    gearchiveerd_procestermijn_onbekend = ChoiceItem(
+        'gearchiveerd_procestermijn_onbekend',
+        _("De zaak cq. het zaakdossier is als geheel niet-wijzigbaar "
+          "bewaarbaar gemaakt maar de vernietigingsdatum kan nog niet bepaald "
+          "worden.")
+    )
+    # After deliberation this element was removed because "vernietigd" means
+    # it's really gone and the status wouldn't make sense:
+    #
+    # vernietigd = ChoiceItem('vernietigd',
+    #     _("De zaak cq. het zaakdossier is vernietigd.")
+    # )
+    overgedragen = ChoiceItem('overgedragen',
+        _("De zaak cq. het zaakdossier is overgebracht naar een "
```

Comment:
PEP8 is setup to allow for 120 char line lengths, so I think this fits a single line